### PR TITLE
[4.2] Load the extension elements from the model in presets download

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
The presets download view for the admin items, should load the extension elements from the model.

### Testing Instructions
- Create a new administrator menu with the preset "Components Dashboard"
- Go to the menu items of this new menu, there should be only one with the name "Components"
- Click on it a couple of components to hide  
image
- Save and close the menu item
- Go mack to the menu manager
- Click on export presets for the new menu  
image

### Actual result BEFORE applying this Pull Request
All Works.

### Expected result AFTER applying this Pull Request
All works.

### CODE REVIEW

**Comment by pe7er** [comment]
I have tested this item :white_check_mark: successfully on 490dccfd4577cb16ff62455e487fcb5140f3a23b<br /><br />The "Download as Preset" keeps working okay. 
The XML exports of the Menu Item from before & after applying the PR, are identical.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37639">issues.joomla.org/tracker/joomla-cms/37639</a>.</sub>

File: administrator/components/com_menus/src/Model/MenuModel.php [hunk 1]
```diff
@@ -391,6 +391,29 @@ public function &getModules()
 		return $result;
 	}
 
+	/**
+	 * Returns the extension elements fir the given items
```
Comment (Quy) [inline administrator/components/com_menus/src/Model/MenuModel.php]:
```suggestion
	 * Returns the extension elements for the given items
```

File: administrator/components/com_menus/src/Model/MenuModel.php [hunk 2]
```diff
@@ -391,6 +391,29 @@ public function &getModules()
 		return $result;
 	}
 
+	/**
+	 * Returns the extension elements for the given items
+	 *
+	 * @param  array  $items  The item ids
```
Comment (Quy) [inline administrator/components/com_menus/src/Model/MenuModel.php]:
```suggestion
	 * @param  array  $itemIds  The item ids
```

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on fc2fe3a47556fc849db76d20149a781aca973915<br /><br />Tested successfully in Joomla 4.2.0-alpha3-dev of 3 May in Wampserver 3.2.8 using PHP 8.0.15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37639">issues.joomla.org/tracker/joomla-cms/37639</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37639">issues.joomla.org/tracker/joomla-cms/37639</a>.</sub>

**Comment by roland-d** [comment]
Thanks everybody

**Pull request merged**